tests/test_driver/test_driver.py

```diff
@@ -0,0 +1,133 @@
+import unittest

```

Comment:
Also, I realize my pylint script wasn't working for tests, so to run the script:

In scripts/lint.py replace line 4 so that it looks like this:
```
lint.py_run(
    "src tests --rcfile pylintrc",
)
```

You should see the things you need to lint under ` ************* Module tests.test_driver.test_driver`